From OmniOS

When running the tests as part of the build, we want to run them in a
consistent order and not automatically re-launch them in verbose mode
when a failure occurs.

diff -wpruN '--exclude=*.orig' a~/Tools/scripts/run_tests.py a/Tools/scripts/run_tests.py
--- a~/Tools/scripts/run_tests.py	1970-01-01 00:00:00
+++ a/Tools/scripts/run_tests.py	1970-01-01 00:00:00
@@ -33,8 +33,6 @@ def main(regrtest_args):
     args.extend(test.support.args_from_interpreter_flags())
 
     args.extend(['-m', 'test',    # Run the test suite
-                 '-r',            # Randomize test order
-                 '-w',            # Re-run failed tests in verbose mode
                  ])
     if sys.platform == 'win32':
         args.append('-n')         # Silence alerts under Windows
